docs/azure_arc_jumpstart/azure_arc_sqlsrv/day2/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+type: docs
+title: "Unified Operations Use Cases"
+linkTitle: "Unified Operations Use Cases"
+weight: 8
```

Comment:
If the intention is for this page to be here, it should be _weight: 5_

![image](https://github.com/microsoft/azure_arc/assets/46730876/0b375a73-7759-4005-a9c4-3e76bffb1f6e)